### ISSUE TITLE
Does not work with Java 18

### ISSUE THREAD

**Issue by premanandc** [ContextMapper/contextmapper.github.io#74]
I get this exception when I try to use the plugin with Java 18 on the path:

```
com.google.inject.internal.util.$ComputationException: com.google.inject.internal.util.$ComputationException: java.lang.ExceptionInInitializerError
```
Switching to a Java 11 SDK seems to work.

**Comment by stefan-ka** [comment]
Thanks for reporting it @premanandc!

I'm actually working on an update that supports newer Java versions... But with the Xtext framework and all the dependencies, this is harder then one might think :) But I'm on it :+1:

**Comment by stefan-ka** [comment]
Hi @premanandc 
Today we released v6.7.0 which finally support Java 17: https://contextmapper.org/news/2022/12/02/v6.7.0-released/
We wanted to have 17 support first, as this is the LTS release of Java. So, I haven't tested Java 18 yet, but I could imagine that it also works with our new release?
Could you maybe test that?
Thanks and best regards, Stefan
